Custom service doesn't respect nested service

### Steps to reproduce
- Create a typescript app
- Create a custom service with typescript that is nested in a folder (such as `v1`)
- get the following compilation error in the `<servicename>class.ts` file
```
server/services/v1/auth-management/auth-management.class.ts:5:21 - error TS2307: Cannot find module '../../app.interface'.
5 import { App } from '../../app.interface';
```

### Expected behavior
Respect nested service folders. Not quite sure how this would work, But I would be willing to look at it in the near future (probably next week) if no one gets to it before then

--- a/generators/writing/templates/src/services/name/name.class.ejs
+++ b/generators/writing/templates/src/services/name/name.class.ejs
@@ -71,7 +71,7 @@ module.exports = moduleExports;
 <%# -%>
 <%# --- if-2 starts below. TYPESCRIPT ---------------------------------------------------------- -%>
 <% if (!isJs) { %>
-import { App } from '../../app.interface'<%- sc %>
+import { App } from '../../<%- subFolderReverse %>app.interface'<%- sc %>
 import { Id, NullableId, Paginated, Params, ServiceMethods, SetupMethod } from '@feathersjs/feathers'<%- sc %>
 <%- insertFragment('imports') %>
 <%- insertFragment('init') %>

